apps/test/unit/codebridge/test-files/predictLevelProperties.json

```diff
@@ -35,7 +33,7 @@
     "allowMultipleAttempts": false,
     "placeholderText": "put your answer here!",
     "freeResponseHeight": 100,
-    "codeEditableAfterSubmit": true
+    "codeEditableAfterSubmit": false
```

Comment:
For my understanding, how likely is it that this value will change in practice once a level is available for students to use?

Reply:
Not very likely. What's more likely to happen is that the value will change through the development of the course which could result in the levelbuilder user seeing unexpected things.